### PR TITLE
added kloudsec.com as a free CDN solution made for programmers

### PR DESCRIPTION
**Pull request by nubela**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,6 +272,7 @@ Table of Contents
 
 ## CDN and Protection
 
+  * [kloudsec.com](https://kloudsec.com) - Minimal CDN platform targeted at programmers. Offers a `One-click Encryption` plugin that automatically provisions/renews LetsEncrypt certs for domains.
```
Comment (ripienaar) [inline README.md]: nice, can you add a little detail on whats available for free?

**Comment by DevWurm** [comment]
Very nice! :thumbsup:

**Comment by nubela** [comment]
Cool, let me know if there's anything else I can help with! :)

**Comment by ripienaar** [comment]
thanks :)

**Pull request merged**